com.nextcloud.desktopclient.nextcloud.metainfo.xml

```diff
@@ -0,0 +1,181 @@
+  <summary xml:lang="ro">Nextcloud client de sincronizare pe desktop</summary>
+  <summary xml:lang="zh_CN">Nextcloud 桌面同步客户端</summary>
+  <summary xml:lang="zh_HK">Nextcloud 桌面版同步客户端</summary>
+  <summary xml:lang="zh_TW">Nextcloud 桌面同步客戶端</summary>
+  <summary xml:lang="es_AR">Cliente de sincronización para escritorio Nextcloud </summary>
+  <summary xml:lang="lt_LT">Nextcloud darbalaukio sinchronizavimo programa</summary>
+  <summary xml:lang="th_TH">Nextcloud ไคลเอนต์ประสานข้อมูลเดสก์ท็อป</summary>
+  <summary xml:lang="es_MX">Cliente de escritorio para  sincronziación de Nextcloud</summary>
+  <summary xml:lang="nb_NO">Nextcloud skrivebordssynkroniseringsklient</summary>
+  <summary xml:lang="nn_NO">Nextcloud klient for å synkronisera frå skrivebord</summary>
+  <summary xml:lang="pt_PT">Nextcloud - Cliente de Sincronização para PC</summary>
+  <summary xml:lang="lb">Nextcloud Desktop Synchronisatioun Client</summary>
+  <description>
+    <p>The Nextcloud desktop client allows you to keep one or more folders full of
+your photos, videos and documents synchronized with your server. Any file you
+add, modify or delete in the synced folders on your desktop or laptop will show
+up, change or disappear on the server and all other connected devices. Thanks
+to the client, you can work with your files even when you are not online!</p>
+  </description>
+  <languages>
+    <lang>en</lang>
+    <lang>oc</lang>
+    <lang>ar</lang>
+    <lang>bg_BG</lang>
+    <lang>ca</lang>
+    <lang>da</lang>
+    <lang>de</lang>
+    <lang>ja_JP</lang>
+    <lang>el</lang>
+    <lang>en_GB</lang>
+    <lang>es</lang>
+    <lang>de_DE</lang>
+    <lang>eu</lang>
+    <lang>fr</lang>
+    <lang>gl</lang>
+    <lang>he</lang>
+    <lang>ia</lang>
+    <lang>id</lang>
+    <lang>is</lang>
+    <lang>it</lang>
+    <lang>ko</lang>
+    <lang>hu_HU</lang>
+    <lang>af_ZA</lang>
+    <lang>nl</lang>
+    <lang>et_EE</lang>
+    <lang>pl</lang>
+    <lang>pt_BR</lang>
+    <lang>cs_CZ</lang>
+    <lang>ru</lang>
+    <lang>sl</lang>
+    <lang>sq</lang>
+    <lang>fi_FI</lang>
+    <lang>sv</lang>
+    <lang>tr</lang>
+    <lang>uk</lang>
+    <lang>ro</lang>
+    <lang>zh_CN</lang>
+    <lang>zh_HK</lang>
+    <lang>zh_TW</lang>
+    <lang>es_AR</lang>
+    <lang>lt_LT</lang>
+    <lang>th_TH</lang>
+    <lang>es_MX</lang>
+    <lang>nb_NO</lang>
+    <lang>nn_NO</lang>
+    <lang>pt_PT</lang>
+    <lang>lb</lang>
+  </languages>
+  <project_license>GPL-2.0+</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.nextcloud.desktopclient.nextcloud.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <caption>The options dialog</caption>
+      <image>https://nextcloud.com/wp-content/themes/next/assets/img/clients/desktop/linux.png?x16328</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://nextcloud.com</url>
+  <releases>
+    <release version="3.1.3" date="2021-02-19"/>
```

Comment:
It looks like the version tag is added to the build in [git2changelog.py](https://github.com/nextcloud/desktop/blob/master/admin/linux/debian/scripts/git2changelog.py), but I don’t know exactly what would be involved in creating the list in XML. Presumably it would involve Python writing directly to XML and reconstructing the version list from scratch, which is a bit beyond me at the moment (though idk I could look into it regardless).

Reply:
thanks i didn't know about `git2changelog.py`.

IMHO the changelog integration is not nessesary and could be added later on.

The main problem is see with this currently is that either someone has to update it manually or it will become out of date on the next release. One simple solution would be to have only a single release in the meta-info file, and set it using templates. So maybe using `@MIRALL_VERSION@` but i am not sure where the date would come from.